video/out/hwdec/hwdec_vulkan.c

```diff
@@ -186,8 +186,8 @@ static void mapper_unmap(struct ra_hwdec_mapper *mapper)
     const AVVulkanFramesContext *vkfc = hwfc->hwctx;;
     AVVkFrame *vkf = p->vkf;
 
-    int num_images;
-    for (num_images = 0; (vkf->img[num_images] != NULL); num_images++);
+    int num_images = 0;
```

Comment:
nit: probably not needed, also you changed here, but not in the other place

Reply:
Oops, that was unintended.